### ISSUE TITLE
 Setting configuration from nuget.exe should use the local nuget.config if available

### ISSUE THREAD

**Issue by ridomin** [NuGet/Home#7867]
Configuring client policies the blog post suggested to use `nuget.exe config -set` commands. When I ran this command in a folder that contains a `nuget.config`  the change should be applied to that file instead of the global one.

Additonally the command should report which file has been updated.

//cc @karann-msft

**Comment by dominoFire** [comment]
Thanks for your suggestion, @ridomin, a couple of questions:

1. Which blog post are you referring to? Maybe this?: https://blog.nuget.org/20181205/Lock-down-your-dependencies-using-configurable-trust-policies.html
2. What would happen with the `-configFile` flag in `nuget.exe config`?

**Comment by karann-msft** [comment]
@dominoFire yes that's the right blogpost. The point is the user shouldn't have to explicitly specify `-configfile` if the currenty directory already has a nuget.config file.

**Comment by zkat** [comment]
This needs an official product decision, @anangaur
